configs/default/FOXE-FOXEERF405_24A.config

```diff
@@ -0,0 +1,131 @@
+#define USE_ACC
+#define USE_ACC_SPI_MPU6000
+#define USE_GYRO
+#define USE_GYRO_SPI_MPU6000
+#define USE_GYRO
+#define USE_GYRO_SPI_ICM42688P
+#define USE_ACC
+#define USE_ACC_SPI_ICM42688P
+#define USE_BARO
+#define USE_BARO_DPS310
+#define USE_FLASH
+#define USE_FLASH_M25P16
+#define USE_MAX7456
+board_name FOXEERF405_24A
```

Comment:
I suggest using FOXEERF405_AIO to be consistent with

![image](https://github.com/betaflight/unified-targets/assets/8344830/bb2717b0-b9c1-4a72-98fd-b9ebbb9cb9da)

Reply:
OK, please delete this PR, I will rename the fw name